_templates/exchanges.html

```diff
@@ -201,7 +205,14 @@ <h3 id="united-kingdom" class="no_toc">United Kingdom</h3>
 <div class="toccontent-block boxexpand expanded">
   <h2 class="exchanges-tab-title exchanges-africa-title" id="africa">Africa</h2>
   <div class="row marketplace-row">
-
+    
+    <div class="general-marketplace">
+      <p>
+        <a class="marketplace-link" href="https://binance.co.ug">Binance Uganda</a>
```

Comment:
The Ugandan flag also needs to be added here, similarly to the nearby examples for Nigeria and South Africa.

Reply:
Okay. So I would need to have a separate pull request for the addition of the Ugandan flag?

Reply:
No, you can do it on this working branch as part of this PR.